examples/ota-requestor-app/esp32/main/main.cpp

```diff
@@ -52,57 +56,24 @@ using namespace ::chip::DeviceLayer;
 
 struct CmdArgs
 {
-    struct arg_str * ipAddr;
-    struct arg_int * nodeId;
     struct arg_end * end;
 };
 
 namespace {
 const char * TAG = "ota-requester-app";
 static DeviceCallbacks EchoCallbacks;
-CmdArgs queryImageCmdArgs, applyUpdateCmdArgs;
-} // namespace
-
-void QueryImageTimerHandler(Layer * systemLayer, void * appState)
-{
-    ESP_LOGI(TAG, "Calling SendQueryImageCommand()");
-    OTARequesterImpl::GetInstance().SendQueryImageCommand(queryImageCmdArgs.ipAddr->sval[0], queryImageCmdArgs.nodeId->ival[0]);
-}
-
-void ApplyUpdateTimerHandler(Layer * systemLayer, void * appState)
-{
-    ESP_LOGI(TAG, "Calling SendApplyUpdateRequestCommand()");
-    OTARequesterImpl::GetInstance().SendApplyUpdateRequestCommand(queryImageCmdArgs.ipAddr->sval[0],
-                                                                  queryImageCmdArgs.nodeId->ival[0]);
-}
-
-int ESPQueryImageCmdHandler(int argc, char ** argv)
-{
-    int nerrors = arg_parse(argc, argv, (void **) &queryImageCmdArgs);
-    if (nerrors != 0)
-    {
-        arg_print_errors(stderr, queryImageCmdArgs.end, argv[0]);
-        return 1;
-    }
-    ESP_LOGI(TAG, "ipAddr:%s nodeId:%x", queryImageCmdArgs.ipAddr->sval[0], queryImageCmdArgs.nodeId->ival[0]);
+CmdArgs applyUpdateCmdArgs;
```

Comment:
Since the query image command is removed, can we also remove the apply image command and use `OTARequestorDriverImpl::ImageDownloadComplete()` to trigger apply image?

Reply:
I didn't find somewhere calling the `ImageDownloadComplete`. so I still reserve the apply image command. Or we can call the `OTAImageProcessorImpl::Apply()` in `HandleFinalize`?

Reply:
Okay, then I guess it is fine to have Apply Image Command.

Reply:
Yes, we have not yet coded the call to `ImageDownloadComplete()` but it will be called eventually.